samples/bluetooth/mesh/light_switch/README.rst

```diff
@@ -130,6 +130,54 @@ FEM support
 
 .. include:: /includes/sample_fem_support.txt
 
+Low Power Node Support
+======================
+
+The mesh light switch sample can also be run as a Low Power Node (LPN). To make the light switch run as a LPN, build the sample using the following command:
+
+  .. code-block:: console
+
+     west build -b <BOARD> -p -- -DOVERLAY_CONFIG="lpn.conf"
+
+The :file:`lpn.conf` is a configuration overlay that will enable the LPN feature, as well as altering certain configuration options to further lower the power consumption. To review the specific alterations, please look at the :file:`lpn.conf`.
+
+While running the sample with the LPN configuration, the fourth Gen. OnOff Client instance will be omitted and the corresponding button will instead be used to temporarily enable GATT proxy on the LPN device. Running GATT proxy continuously will consume considerable power, and therefore the application is set to automatically disable GATT proxy advertisement after a period of 30 seconds, given that the device is provisioned. In this way the user will be able to manually re-enable GATT proxy for a short period by pressing button 4 on the device, and can then connect over GATT and perform necessary configuration. When the configuration is complete the user will then disconnect from the LPN and the GATT proxy advertisement will stop.
```

Comment:
(nRF Mesh application) For iPhone users: disable Automatic Connection under Proxy. The user will then be able to use the Disconnect option. The user must change tabs before seeing that the proxy is unselected.

Reply:
I've added the note about disabling auto-connection. The note regarding changing tabs should perhaps be reported to the mobile apps team.